src/entity/User.ts

```diff
@@ -14,6 +14,6 @@ export class User {
   @Column()
   password: string;
 
-  @Column()
+  @Column({ nullable: true })
   birthDate: string;
```

Comment:
Quando um campo é nullable, é legal marcamos como opcionalno Typescript:

```suggestion
  @Column({ nullable: true })
  birthDate?: string;
```